source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po

```diff
@@ -1125,7 +1126,7 @@ msgstr ">=0 (absolute waarde)"
 
 #: ../../chapters/appendix/mixxx_controls.rst:404
 msgid "Latency slider in the prefs"
-msgstr "Latency glijder in the voorkeuren"
+msgstr ":term:`Latentie glijder in the voorkeuren"
```

Comment:
```suggestion
msgstr ":term:`Latentie <latency>` glijder in the voorkeuren"
```

Reply:
in Dutch we speak about latentie, I mentioned the English word so readers can get acquainted with the English terms as well.